datadog/datachecks.q

```diff
@@ -0,0 +1,51 @@
+//DATADOG CHECKS
+//Script for running checks manually/via crontab
+//Makes dictionary of defaults and uses .Q.opt to refer to command values passed by its key.
+o:.Q.def[`user`pass`timeout`init`noexit!(`admin;`admin;100;1b;0b);.Q.opt[.z.x]]
```

Comment:
:eyes:
This not making use of TorQ input handling?

Reply:
At the moment this is run through a crontab not with a TorQ process.